src/components/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ClassNames from 'classnames';
+
+import './Tabs.scss';
+
+export class Tabs extends React.Component {
+  state = {
+    index: 0,
+  }
+
+  render() {
+    const { index } = this.state;
+    const { tabSelected, API } = this.props;
+
+    return (
+      <>
+        <nav className="tabs">
+          <button
+            data-index="1"
+            className={ClassNames(
+              'tabs__button',
+              { ' tabs__button-selected': index === 1 },
+            )}
+            type="button"
+            onClick={(event) => {
+              this.setState({
+                index: 1,
+              });
+              tabSelected(API.getAll);
```

Comment:
```suggestion
            onClick={() => {
              this.setState({
                index: 1,
              });
              tabSelected(API.getAll);
```
you don't use event?